azure.yaml

```diff
@@ -14,7 +14,7 @@
       version: 1.2.0
     - app: kube-state-metrics
       componentVersion: 1.9.2
-      version: 1.0.2
+      version: 1.0.3
```

Comment:
IMO this shouldn't be done - changing the definition of already published release.

@wtty-fool would it make sense to keep this PR only for adding 1.0.3 kube-state-metrics to the apps changelog? We're in the process of releasing next patch releases (see other PRs in the repo), would be happy to include kube-state-metrics v1.0.3 in them

Reply:
Thanks @sslavic for spotting that. Then we can include that only for AWS as this is apparently the only provider with WIP release in place... Is that ok?

Reply:
Sure, once you merge I'll take it into separate already created PR which activates that release and makes more changes to the release definition/package to ship them all together as one.

Reply:
@sslavic Good catch, thank you for pointing that out.
Ok, I'll keep an update to AWS and changelog, the rest will be dropped from this PR.